.github/scripts/get_release_version.py

```diff
@@ -62,7 +62,8 @@
 
 with open(os.getenv("GITHUB_ENV"), "a") as githubEnv:
     if gitRef is None:
-        print("This is not running in github, GITHUB_REF is null. Assuming a local build...")
+        print(
+            "This is not running in github, GITHUB_REF is null. Assuming a local build...")
```

Comment:
original code seems more readable and it is not really getting into multi lines

Reply:
I think this is how a python file gets reformatted.

Reply:
after a certain character count in a given line